HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+# CMSSW version specific customizations
+import os
+cmsswVersion = os.environ['CMSSW_VERSION']
+
+if cmsswVersion >= "CMSSW_7_5":
+
```

Comment:
No, this file will be used in all releases (74/75/76/...), to have a uniform treatment.
Only right now it contains just 75 stuff...